packages/app/styles/global.css

```diff
@@ -30,3 +30,24 @@ body {
   border-color: #3684c6;
   background-color: #3684c6;
 }
+
+.ant-modal-body {
+  padding: 0px 24px;
+}
```

Comment:
Couldn't figure out how to get this css inside the `ReleaseNotes.tsx` file so that it doesn't affect the other modals we have. Any help would be appreciated

Reply:
can't you just inline style it for now? or styled-component it? I guess the question is -- how come those *aren't* viable options?

Reply:
I didn't know if it was possible to do it inline or with a styled-component because the `.ant-modal-body` is a subclass of the modal but I don't know much about styling so I probably just don't know how to do it..

Reply:
should be able to move this CSS into the `bodyStyle` property of the modal: https://ant.design/components/modal/#API

Reply:
that fixes it, thanks!